src/extensions/default/bramble/lib/PostMessageTransport.js

```diff
@@ -93,11 +93,25 @@ define(function (require, exports, module) {
             }
 
             if(ConsoleManager.isConsoleRequest(msgObj.message)) {
+                if(msgObj.data.type === "error-handler"){
+                    var regex = new RegExp('(blob:.+):([^:]+):(.+)', 'gm');
+                    var endingRegex = new RegExp(':([0-9]+):([0-9]+)$', 'gm');
+
+                    var stackTrace = msgObj.data.args["Stack"].split("@");
+                    msgObj.data.args = [];
+
+                    // Handle Blob URLs
+                    for(var i = 1; i < stackTrace.length; i++){
+                        var stackItem = stackTrace[i].match(regex);
+                        stackItem = (BlobUtils.getFilename(stackItem[0].replace(endingRegex, ''))).split('/');
+                        msgObj.data.args.push('Function Line:' + stackItem[1] + ' File:' + stackItem[stackItem.length -1 ]);
+                    }
```

Comment:
Move all this code for regex'ing the message into your `ConsoleManager` module.  Just call `ConsoleManager.handleConsoleRequest(msgObj.data);` and then do it in there (or call a function in there).  Let's keep this file about routing messages from the iframe.